web/pdf_page_view.js

```diff
@@ -69,6 +71,7 @@ var PDFPageView = (function PDFPageViewClosure() {
     var renderingQueue = options.renderingQueue;
     var textLayerFactory = options.textLayerFactory;
     var annotationLayerFactory = options.annotationLayerFactory;
+    var enhanceTextSelection = options.enhanceTextSelection || false;
```

Comment:
The intention is to enable this by default (see previous comments), so let's change this to `true` (and in the comment above).